include/libopencm3/efm32/hg/cmu.h

```diff
@@ -633,6 +634,7 @@ enum cmu_osc {
 	LFXO, /**< External, 32.768kHz */
 	AUXHFRCO, /**< Internal, 1-28Mhz */
 	USHFRCO, /**< Internal, 48MHz */
+	USHFRCODIV2, /**< Internal, 24MHz */
```

Comment:
I don't really think this should be treated as a separate oscillator.  I think the comment above should be changed, but normally, and on other platforms, these optionally divided clocks are treated as dividers in the clock configs, not separate clocks.

Reply:
But I also see that it's a command option in the HFCLKSEL.  *shrugs*  I'll look at the rest of the patch.  refman is a little weird here maybe.

Reply:
The HFCLKSEL can take one of 5 values: `HFRCO`, `HFXCO`, `LFRCO`, `LFXCO`, and `USHFRCODIV2`.  `USHFRCO` is not one of the values.

How do you think this should be handled in `cmu_set_hfclk_source()` if `USHFRCODIV2` is not given an enum in `cmu_osc`?

Reply:
looking at it again, should we just drop the USHFRCO from the list? would that not be most self consistent?

Reply:
nah, that's wrong too.  Hoenstly, I think the problem is trying to use the same enumeration for oscillators on/off, and _also_ for "cpu clock sources"

Reply:
nah, I think that's even more confusing :)  I don't really like it, but I don't see anything better at the moment, and the clock tree for the HG is actually fairly simply really, so I'll probably just pull this as is and see what turns out in the future